ingestion-beam/src/test/java/com/mozilla/telemetry/util/NormalizeTest.java

```diff
@@ -0,0 +1,47 @@
+package com.mozilla.telemetry.util;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class NormalizeTest {
+
+  @Test
+  /**
+   * See a selection of existing raw channel names and their normalized versions in BigQuery:
+   *
+   * <pre>{@code
+   * SELECT channel, normalized_channel, count(*)
+   * FROM `moz-fx-data-derived-datasets.telemetry.clients_daily_v6`
+   * WHERE true
+   *   and submission_date_s3 >= DATE('2018-07-01')
+   *   and normalized_channel != 'Other'
+   * group by 1, 2
+   * order by 3 desc
+   * LIMIT 1000
+   * }</pre>
+   */
+  public void channel() {
+    assertEquals("release", Normalize.channel("release"));
+    assertEquals("esr", Normalize.channel("esr"));
+    assertEquals("beta", Normalize.channel("beta"));
+    assertEquals("aurora", Normalize.channel("aurora"));
+    assertEquals("nightly", Normalize.channel("nightly"));
+
+    assertEquals("nightly", Normalize.channel("nightly-cck-ubuntu"));
```

Comment:
@trink @whd 

There is a special case in the lua implementation to handle "nightly-cck-" prefix. Is that still needed? Do we have documentation of the significance of that case?

Reply:
I believe [CCK](https://mike.kaply.com/2012/04/13/customizing-firefox-extensions-and-the-cck-wizard/) is for pre-quantum and potentially ESR enterprise deployments of firefox. We might still care about these, though I believe ESR is quantum by now and so these builds wouldn't be supported anymore. @mreid-moz or similar can make a policy decision, but they are probably a sufficiently small number of pings as to be irrelevant how we bucket them (though again, data would be better).

Reply:
I'll plan to add a comment explaining what `-cck` means and keep the current behavior.